sretoolbox/container/image.py

```diff
@@ -113,18 +127,34 @@ def __init__(self, url, tag_override=None, username=None, password=None,
 
         self._cache_tags = None
         self._cache_manifest = None
+        self._cache_content_type = None
+
+    @property
+    def content_type(self):
+        """
+        Property to return the Content-Type header from the manifest retrieval.
+        It caches the result.
+        """
+        if self._cache_content_type is None:
+            _ = self.manifest
+
+        if self._cache_content_type is NO_SUCH_HEADER:
+            raise HTTPError('Content-Type header not found.')
+
+        return self._cache_content_type
 
     @property
     def digest(self):
         """
-        Property to cache the digest returned by get_manifest()
+        Property to return the Docker-Content-Digest header from the manifest
+        retrieval. It caches the result.
         """
         if self._cache_digest is None:
-            manifest = self._get_manifest()
-            digest = manifest.headers.get('Docker-Content-Digest')
-            if digest is None:
-                raise HTTPError('Docker-Content-Digest header not found.')
-            self._cache_digest = digest
+            _ = self.manifest
+
+        if self._cache_digest is NO_SUCH_HEADER:
```

Comment:
same... just check for `None` again

Reply:
fixed in https://github.com/app-sre/sretoolbox/commit/b58b09be4a6c06a486afa871c6da3e810f22df98